lib/tpm2_kdfe.c

```diff
@@ -0,0 +1,238 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <errno.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <openssl/bn.h>
+#include <openssl/ecdh.h>
+
+#include <tss2/tss2_tpm2_types.h>
+#include <tss2/tss2_mu.h>
+
+#include "log.h"
+#include "tpm2_openssl.h"
+#include "tpm2_alg_util.h"
+#include "tpm2_util.h"
+
+TSS2_RC tpm2_kdfe(
+        TPMI_ALG_HASH hashAlg,
```

Comment:
@t-higuchi, do not specify each argument on a new line. Align these arguments at 80 character width.